tests/integration/snowflake/test_snowflake_adapter.py

```diff
@@ -0,0 +1,347 @@
+import urllib.parse
+from random import randrange
+
+import pytest
+import yaml
+
+import snowshu.core.models.materializations as mz
+from snowshu.adapters.source_adapters import BaseSourceAdapter
+from snowshu.adapters.source_adapters.snowflake_adapter import SnowflakeAdapter
+from snowshu.core.models.credentials import Credentials
+from snowshu.core.models.relation import Relation
+from snowshu.exceptions import TooManyRecords
+from snowshu.samplings.sample_methods import BernoulliSampleMethod
+from tests.assets.integration_test_setup import CREDENTIALS, get_connection_profile
+from tests.common import query_equalize, rand_string
+
+
+@pytest.fixture(scope='session')
+def sf_adapter():
+    with open(CREDENTIALS) as cred_file:
+        credentials = yaml.safe_load(cred_file)
+
+    profile_dict = get_connection_profile(credentials)
+    adapter = SnowflakeAdapter()
+    adapter.credentials = Credentials(**profile_dict)
+
+    return adapter
+
+
+def test_directionally_wrap_statement(sf_adapter):
+    sampling = BernoulliSampleMethod(50, units='probability')
+    query = """SELECT * FROM "SNOWSHU_DEVELOPMENT"."EXTERNAL_DATA"."ADDRESS_REGION_ATTRIBUTES"
+            WHERE IS_CURRENTLY_TARGETED = TRUE
+            AND SALES_REGION IN ('northeast', 'southeast')
+            AND PRIMARY_REGIONAL_CREDIT_PROVIDER = 'mastercard'"""
+    DATABASE, SCHEMA, TABLE = "SNOWSHU_DEVELOPMENT", "EXTERNAL_DATA", "ADDRESS_REGION_ATTRIBUTES"
+    relation = Relation(database=DATABASE,
+                        schema=SCHEMA,
+                        name=TABLE,
+                        materialization=TABLE,
```

Comment:
Sorry I missed this on the first pass, but the `materialization` attribute should be one of the materialization types created [here](https://github.com/Health-Union/snowshu/blob/ea0bf0eaa82a3d2834262025f19d3da6ab6027e4/snowshu/core/models/materializations.py#L5).

However, it might not be required for most of the relations here and could be left out entirely.